test/cases/db-update.bats

```diff
@@ -222,7 +222,7 @@ load ../lib/common
 
 @test "package has to be aregular file" {
 	local p
-	local target=$(mktemp -d)
+	local target=$(mktemp -dt)
```

Comment:
No TEMPLATE given, this change is extraneous.

Reply:
Whoops, you are correct.